.github/workflows/add_release_label.yml

```diff
@@ -0,0 +1,62 @@
+name: Add release label to PR and linked issues when PR gets merged
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - closed
+
+jobs:
+  add_release_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0 # This is needed to checkout all branches
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Get the next semver version
+        id: get-next-semver-version
+        run: |
+          # Get the highest version from release branches
+          VERSION_BRANCHES=$(git branch -r | grep -o 'Version-v[0-9]*\.[0-9]*\.[0-9]*' | grep -o '[0-9]*\.[0-9]*\.[0-9]*' | sort --version-sort | tail -n 1)
```

Comment:
This should be quoted to prevent word splitting

I'd recommend running this script through https://www.shellcheck.net/ to ensure it passes that linter. We use Shellcheck for linting our Bash scripts, but it misses these scripts that are inlined in workflows

Reply:
I see that it gets confused by the references to variables; perhaps those could be set as environment variables instead.

Reply:
This is now fixed with this [commit](https://github.com/MetaMask/metamask-extension/pull/19061/commits/9a20d5abe56d3c5cfe93cca6a44c2601ba4447e3).